libs/wingsdk/.projen/deps.json

```diff
@@ -167,57 +167,56 @@
     },
     {
       "name": "@aws-sdk/client-cloudwatch-logs",
-      "version": "3.449.0",
+      "version": "3.515.0",
       "type": "bundled"
     },
     {
       "name": "@aws-sdk/client-dynamodb",
-      "version": "3.449.0",
+      "version": "3.515.0",
       "type": "bundled"
     },
     {
       "name": "@aws-sdk/client-elasticache",
-      "version": "3.449.0",
+      "version": "3.515.0",
       "type": "bundled"
     },
     {
       "name": "@aws-sdk/client-lambda",
-      "version": "3.449.0",
+      "version": "3.515.0",
       "type": "bundled"
     },
     {
       "name": "@aws-sdk/client-s3",
-      "version": "3.449.0",
+      "version": "3.515.0",
       "type": "bundled"
     },
     {
       "name": "@aws-sdk/client-secrets-manager",
-      "version": "3.449.0",
+      "version": "3.515.0",
       "type": "bundled"
     },
     {
       "name": "@aws-sdk/client-sns",
-      "version": "3.449.0",
+      "version": "3.515.0",
       "type": "bundled"
     },
     {
       "name": "@aws-sdk/client-sqs",
-      "version": "3.449.0",
+      "version": "3.515.0",
       "type": "bundled"
     },
     {
       "name": "@aws-sdk/s3-request-presigner",
-      "version": "3.449.0",
+      "version": "3.515.0",
       "type": "bundled"
     },
     {
       "name": "@aws-sdk/types",
-      "version": "3.449.0",
```

Comment:
why do you skip this?

Reply:
It's in the notes: https://github.com/winglang/wing/pull/5429/files#diff-0769c0418cbdd805ac163690fe215884e186a895522b0e382e80b992a0eb3487R150-R151

Reply:
It's in the comment: https://github.com/winglang/wing/pull/5429/files#diff-0769c0418cbdd805ac163690fe215884e186a895522b0e382e80b992a0eb3487R150-R151